hwilib/devices/bitbox02.py

```diff
@@ -0,0 +1,678 @@
+from typing import (
+    cast,
+    Any,
+    Callable,
+    Dict,
+    Optional,
+    Union,
+    Tuple,
+    List,
+    Sequence,
+    TypeVar,
+)
+from binascii import unhexlify
+import struct
+import builtins
+import sys
+from functools import wraps
+
+from ..hwwclient import HardwareWalletClient
+from ..serializations import (
+    PSBT,
+    CTxOut,
+    is_p2pkh,
+    is_p2wpkh,
+    is_p2wsh,
+    ser_uint256,
+    ser_sig_der,
+)
+from ..errors import (
+    HWWError,
+    ActionCanceledError,
+    BadArgumentError,
+    DeviceNotReadyError,
+    UnavailableActionError,
+    DEVICE_NOT_INITIALIZED,
+    handle_errors,
+    common_err_msgs,
+)
+
+import hid  # type: ignore
+
+from .trezorlib.tools import parse_path
```

Comment:
I think we should avoid doing cross device imports like this. In theory, bitbox02 support should still work even if all of the trezor stuff were deleted.

It might be useful to add some of our own utility functions like this that all devices can share.

Reply:
Good point. Fixed in a new commit. I can factor out common functionality in a future PR.